projects/storefrontlib/src/lib/ui/pages/order-confirmation-page/order-confirmation-page.module.ts

```diff
@@ -16,7 +17,7 @@ const routes: Routes = [
     path: null,
     canActivate: [AuthGuard, CmsPageGuards, OrderConfirmationPageGuard],
     component: OrderConfirmationPageComponent,
-    data: { pageLabel: 'cartPage', cxPath: 'orderConfirmation' }
+    data: { pageLabel: 'orderConfirmationPage', cxPath: 'orderConfirmation' }
```

Comment:
can we add a TODO that as soon as the components are moved to CMS driven components we can drop this specific `OrderConfirmationPageComponent`?

Reply:
Done